src/sst/core/main.cc

```diff
@@ -426,6 +426,9 @@ main(int argc, char *argv[])
     double start_graph_gen = sst_get_cpu_time();
     graph = new ConfigGraph();
 
+    // Need to initialize TimeLord before we use UnitAlgebra
+    Simulation::getTimeLord()->init(cfg.timeBase);
```

Comment:
This breaks macro here:
````python
  sst.setProgramOption("timebase", "100as")
````
Because you're now setting the timebase before the config graph has been processed, elements can no longer set program options. Later on, we create links with "100as" timebase, which now crashes because the timebase is always "1ps" no matter what options you set in your python file.